.github/workflows/main.yml

```diff
@@ -0,0 +1,27 @@
+name: Tests
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-16.04, ubuntu-18.04]
```

Comment:
Cool !, However, why you are running it on different operating systems instead of different go versions ? or do we need it, let's talk about it at meeting.